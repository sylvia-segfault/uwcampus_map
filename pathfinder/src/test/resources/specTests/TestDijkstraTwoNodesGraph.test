# Find a path with Dijkstra's algorithm on
# a graph that contains two nodes which are
# connected by two different edge labels

CreateGraph graph
AddNode graph A
AddNode graph B
AddEdge graph A B 10.0
AddEdge graph A B 5.0
FindPath graph A B