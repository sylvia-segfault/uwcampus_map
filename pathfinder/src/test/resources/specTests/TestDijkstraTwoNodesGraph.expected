# Find a path with Dijkstra's algorithm on
# a graph that contains two nodes which are
# connected by two different edge labels

created graph graph
added node A to graph
added node B to graph
added edge 10.000 from A to B in graph
added edge 5.000 from A to B in graph
path from A to B:
A to B with weight 5.000
total cost: 5.000